app/utils/simplified-account/settings-builder.js

```diff
@@ -0,0 +1,124 @@
+  const aboutYourServiceSettings = [
+    {
+      id: 'service-name',
+      name: 'service name',
+      url: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.serviceName.index, account.service_id, account.type),
+      current: currentUrl.includes('simplified') && currentUrl.includes('settings/service-name'),
+      permitted: permissions.service_name_update,
+      alwaysViewable: false
+    },
+    {
+      id: 'email-notifications',
+      name: 'email notifications',
+      url: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.emailNotifications.index, account.service_id, account.type),
+      current: currentUrl.includes('simplified') && currentUrl.includes('settings/email-notifications'),
+      permitted: true, // TODO
+      alwaysViewable: true
+    },
+    {
+      id: 'team-members',
+      name: 'team members',
+      url: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.index, account.service_id, account.type),
+      current: currentUrl.includes('simplified') && currentUrl.includes('settings/team-members'),
+      permitted: true, // TODO
+      alwaysViewable: false
+    },
+    {
+      id: 'org-details',
+      name: 'Organisation details',
+      url: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.index, account.service_id, account.type),
+      current: currentUrl.includes('simplified') && currentUrl.includes('settings/org-details'),
+      permitted: true, // TODO
+      alwaysViewable: false
+    }
+  ]
+  aboutYourServiceSettings.forEach(setting => serviceSettings['about your service'].push(setting))
+  return serviceSettings
+}
+
+const buildPaymentSettings = (serviceSettings, account, currentUrl) => {
+  const paymentSettings = [
+    {
+      id: 'card-paymemts',
+      name: 'card payments',
+      url: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.index, account.service_id, account.type),
+      current: currentUrl.includes('simplified') && currentUrl.includes('settings/team-members'),
+      permitted: true, // TODO
+      alwaysViewable: true
+    },
+    {
+      id: 'card-types',
+      name: 'card types',
+      url: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.index, account.service_id, account.type),
+      current: currentUrl.includes('simplified') && currentUrl.includes('settings/team-members'),
+      permitted: true, // TODO
+      alwaysViewable: true
+    }
+  ]
+  paymentSettings.forEach(setting => serviceSettings['payments'].push(setting))
+  return serviceSettings
+}
+
+const buildDeveloperSettings = (serviceSettings, account, currentUrl) => {
+  serviceSettings['developers'].push({
+    id: 'api-keys',
+    name: 'API keys',
+    url: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.index, account.service_id, account.type),
+    current: currentUrl.includes('simplified') && currentUrl.includes('settings/team-members'),
+    permitted: true, // TODO
```

Comment:
There's a TODO here, is that something that should be resolved before merging?

Reply:
no, this PR scaffolds quite a bit of work yet to come. Any TODOs are to serve as 'helpful' reminders